src/style.css

```diff
@@ -1 +1,91 @@
-/* add styles here */
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir;
+  src:
+    url(./fonts/Avenir.ttc) format('truetype'),
+    url(./fonts/Avenir-Book.ttf) format('truetype'),
+    url(./fonts/Avenir-Heavy.ttf) format('truetype');
+}
+
+html {
+  font-weight: 300;
+}
+
+.body {
+  margin-top: 0;
+}
+
+.input {
+  display: flex;
+  flex-direction: column;
+}
+
+.input__state {
+  font-family: Avenir, Arial, sans-serif;
+  color: #3d4e61;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  border-radius: 4px;
+  padding: 0;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+}
+
+.input__state::placeholder {
+  color: #3d4e61;
+}
+
+.input--big {
+  margin: 20px 0;
```

Comment:
it is better not to use both margin top and bottom together

![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/108538921/ea4cf131-c741-4957-98b4-4aa764dcc78b)